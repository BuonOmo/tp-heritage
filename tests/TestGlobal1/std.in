S seg0 75 -49 -928 -794
R rec0 -868 538 805 560
S seg1 -251 -11 587 220
R rec1 -740 802 104 -827
R rec2 -175 952 -400 -20
S seg2 102 439 808 -892
S seg3 -230 -164 -794 -443
S seg4 -368 -362 371 121
S seg5 -914 -267 668 880
R rec3 416 284 -682 -610
R rec4 -501 -699 -236 188
R rec5 -514 886 784 -5
R rec6 813 233 -57 -316
S seg6 181 -945 -644 -551
R rec7 973 -786 983 226
R rec8 -323 507 475 326
S seg7 -551 -686 -919 -689
S seg8 590 312 641 915
S seg9 -970 -308 -890 349
S seg10 260 -622 759 -696
R rec9 298 -946 -761 90
S seg11 75 -993 -515 92
R rec10 890 -228 801 -921
R rec11 -466 -316 416 -389
S seg12 -510 254 -845 787
S seg13 -617 396 -865 820
S seg14 -766 533 -452 -559
S seg15 -615 336 283 888
S seg16 -179 -70 804 722
S seg17 -123 -353 -386 -8
R rec12 431 -479 820 590
S seg18 -440 154 -568 -80
R rec13 -417 785 374 275
S seg19 -590 305 -329 -354
S seg20 -55 -877 52 451
S seg21 87 511 859 794
S seg22 -799 -209 945 616
R rec14 854 258 -182 -734
R rec15 -330 914 -897 -281
S seg23 -503 -332 -821 272
R rec16 427 633 -607 -27
S seg24 -217 25 678 -534
S seg25 337 877 -304 -374
R rec17 -127 -686 -408 -389
R rec18 336 55 -77 -9
PC pol0 -726 113 -570 932 -923 752 -696 262
PC pol1 -572 -117 904 988 -957 -24
PC pol2 -237 267 180 -110 968 88 -392 444 355 -972 108 691
PC pol3 -311 223 -468 702 -260 816 522 893
OI int1 rec2 rec2 pol1 seg25
OI int2 int1 seg3 pol3
OR uni1 int2 int1 seg14
OR uni2 uni1 int1
DELETE int1 int2
MOVE uni1 12 54
OI int1 uni2 rec16
HIT int2 0 0
HIT int1 0 0
OR uni3 seg1 seg2 seg3 seg4
OR uni4 pol1 pol2 pol3
OR uni5 rec0 rec1 rec2 rec3 rec4 rec5 rec6 rec7 rec8 rec9 rec10 rec11 rec12 rec13 rec14 rec15 rec16
OR uni6 uni3 uni4 uni5
HIT uni6 0 0
EXIT
