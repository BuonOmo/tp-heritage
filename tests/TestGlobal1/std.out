OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
ERR
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
ERR
NO
OK
OK
OK
OK
YES
